server/controllers/rpaFrameworkCommandsController.js

```diff
@@ -1,25 +1,29 @@
-const activityDataRetrieval = require('../services/ActivityDataRetrieval/ActivityDataRetrieval');
+const mongoose = require('mongoose');
+const rpaModels = require('../models/rpaTaskModel');
 
+// GET /rpa-framework/commands/get-available-applications
 exports.getAvailableApplications = async (req, res) => {
   try {
-    const listOfDistinctApplications = await activityDataRetrieval.getDistinctApplicationsFromDB();
     res.set('Content-Type', 'application/json');
-    res.send(listOfDistinctApplications);
+    mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });    
```

Comment:
If I'm not completely wrong, it's absolutely sufficient to establish the connection to the DB once at startup, i.e., in `server.js`

Reply:
I aggree with Maximilian on this one as my design decision for the server side was inspired by these tutorials ([part1](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose) & [part2](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes)) and they do it differently. But perhaps there is a good reason to do it like this. If so what is the reason?